.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+name: Integration tests
+on: [pull_request]
+jobs:
+  integration_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        mongodb-version: ['7']
+
+    steps:
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.10.0
+        with:
+          mongodb-version: ${{ matrix.mongodb-version }}
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Conda and install python
+        uses: s-weigand/setup-conda@v1
+        with:
+          activate-conda: true
+          python-version: 3.11
```

Comment:
Will you update the software to this version as well?

Reply:
You mean the instructions or the Dockerfile?

Reply:
Yes, so the actions and Dockerfile align

Reply:
Ah OK, good idea, but I'll do it in another PR if it's OK

Reply:
Perfect 👍